catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ValidateCommand.java

```diff
@@ -84,19 +84,21 @@
     @Reference
     CatalogFramework catalog;
 
+    ValidatePrinter printer = new ValidatePrinter();
```

Comment:
Instead of making this package private, we should take that in the constructor or have a setter method to set it and inject this using blueprint.